app/controllers/EventSummaryController.scala

```diff
@@ -94,8 +95,8 @@ class EventSummaryController @Inject()(
       form.bindFromRequest().fold(
         formWithErrors => summaryListRows(waypoints).map(rows => BadRequest(view(formWithErrors, waypoints, rows, selectedTaxYear))),
         value => {
-          val userAnswerUpdated = UserAnswers().setOrException(EventSummaryPage, value)
-          Future.successful(Redirect(EventSummaryPage.navigate(waypoints, userAnswerUpdated, userAnswerUpdated).route))
+          val ua = UserAnswers().setOrException(EventSummaryPage, value)
+          Future.successful(Redirect(EventSummaryPage.navigate(waypoints, ua, ua).route))
```

Comment:
The first UA should be the user answers before the value has been saved, strictly speaking.